modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -191,6 +193,17 @@ object NewPullRequestData {
       change <- SemVer.getChange(curr, next)
     } yield s"semver-${change.render}"
 
+  def updateHasScalaDependency(update: Update): Boolean =
+    update match {
+      case s: Update.Single =>
+        FilterAlg.isScalaDependency(s.groupId.value, s.artifactId.name)
+      case g: Update.Group =>
+        g.crossDependencies
+          .exists(crossDependency =>
+            FilterAlg.isScalaDependency(g.groupId.value, crossDependency.head.artifactId.name)
+          )
+    }
```

Comment:
I think this could be simplified to `update.dependencies.exists(FilterAlg.isScalaDependency)`.

Reply:
thanks for that, simplifies a lot of code